blob/service.go

```diff
@@ -131,6 +132,8 @@ func (s *Service) GetAll(ctx context.Context, height uint64, namespaces []share.
 				resultErr[i] = fmt.Errorf("getting blobs for namespace(%s): %s", namespace.String(), err)
 				return
 			}
+
+			log.Debugw("receiving blobs", "height", height, "blobs", blobs)
```

Comment:
[nit]
I'd also suggest to include the received amount of blobs in the requested namespace.

Reply:
maybe "received %d blobs"?

Reply:
Yeah, smth like that.

Reply:
added a new property called `total` with the number of received blobs